scripts/i2i/translate_image.py

```diff
@@ -16,50 +18,26 @@ def preprocess_mri(x):
     return x
 
 
-def main():
-    parser = argparse.ArgumentParser(description="Translate image.")
-    parser.add_argument(
-        "-m",
```

Comment:
@pcrespov is there a way I can add alias arguments using typer? I kind of like having a shorthand notation. I tried using [decorators](https://stackoverflow.com/questions/29374425/aliasing-in-the-names-of-function-arguments-double-naming/29374525#29374525) but typer ignored it.

Reply:
Maybe: https://stackoverflow.com/questions/50499340/specify-options-and-arguments-dynamically ?

Reply:
```
model: Path = typer.Option(
        ..., "--model", "-m", help="Model file path in .pth format"
    )
```